ui/app/AppLayouts/Communities/panels/MintTokensSettingsPanel.qml

```diff
@@ -598,23 +608,79 @@ StackView {
                 feeErrorText: selfDestructFeesSubscriber.feeErrorText
                 isFeeLoading: !selfDestructFeesSubscriber.feesResponse
 
-                function openPopup(type, userName, address) {
+                function openPopup(type, userName, address, contactId) {
                     tokenMasterActionPopup.actionType = type
-                    tokenMasterActionPopup.userName = userName
+                    tokenMasterActionPopup.userName = userName ||
+                            SQUtils.Utils.elideAndFormatWalletAddress(address)
                     tokenMasterActionPopup.address = address
+                    tokenMasterActionPopup.contactId = contactId
                     open()
                 }
 
+                onRemotelyDestructClicked: signPopup.open()
+                onKickClicked: signPopup.open()
+                onBanClicked: signPopup.open()
+
                 SelfDestructFeesSubscriber {
                     id: selfDestructFeesSubscriber
+
                     walletsAndAmounts: [{ 
                         walletAddress: tokenMasterActionPopup.address, 
-                        amount: 1 
+                        amount: 1
                     }]
                     accountAddress: tokenMasterActionPopup.selectedAccount
                     tokenKey: view.token.key
                     enabled: tokenMasterActionPopup.opened
-                    Component.onCompleted: root.registerSelfDestructFeesSubscriber(selfDestructFeesSubscriber)
+                    Component.onCompleted: root.registerSelfDestructFeesSubscriber(
+                                               selfDestructFeesSubscriber)
+                }
+
+                SignMultiTokenTransactionsPopup {
```

Comment:
Isn't this what you're removing in your other PR? :)

Reply:
No, no, this one is just renamed in other pr. So I will have conflict with myself :)

Reply:
I thought so :)